misk-testing/src/main/kotlin/misk/testing/MiskTestExtension.kt

```diff
@@ -101,6 +121,20 @@ internal class MiskTestExtension : BeforeEachCallback, AfterEachCallback {
       beforeEachCallbacks.forEach { it.beforeEach(context) }
     }
   }
+
+  private fun ExternalDependency.startOnce() {
```

Comment:
super nit: `startIfNecessary` so it's clear it doesn't blow up

Reply:
hmm... that's how I usually interpret `once()` in a function name. Will change though

Reply:
yea, super nit :)